src/Providers/AbstractServiceProvider.php

```diff
@@ -215,7 +215,9 @@ protected function registerManager()
             );
 
             return $instance->setBlacklistEnabled((bool) $this->config('blacklist_enabled'))
-                            ->setPersistentClaims($this->config('persistent_claims'));
+                ->setPersistentClaims($this->config('persistent_claims'));
```

Comment:
This line needs to be deleted.

Reply:
Which line exactly? 218?

Reply:
Exactly. When you have a look at the file https://github.com/PHP-Open-Source-Saver/jwt-auth/blob/54198a59e04e2da50fef49033edd97e1719ec141/src/Providers/AbstractServiceProvider.php#L218 https://github.com/PHP-Open-Source-Saver/jwt-auth/blob/54198a59e04e2da50fef49033edd97e1719ec141/src/Providers/AbstractServiceProvider.php#L219 you will see, that the line exists two times. One with semicolon and one without.